core/modules/utils/dom/scroller.js

```diff
@@ -74,7 +74,7 @@ PageScroller.prototype.scrollIntoView = function(element,callback,options) {
 	this.cancelScroll(srcWindow);
 	this.startTime = Date.now();
 	// Get the height of any position:fixed toolbars
-	var toolbar = srcWindow.document.querySelector(".tc-adjust-top-of-scroll"),
+	var toolbar = $tw.utils.querySelectorSafe(".tc-adjust-top-of-scroll",srcWindow.document),
```

Comment:
I am not sure that we need to use the `querySelectorSafe` method where we are providing the selector since we know that it is valid, though I admit that it cannot hurt. Recommend we get input from others on this.

Reply:
I did think so, too, initially, but consistency won out in the end. At least for me. Other opinions are welcome, of course. Which "others" did you have in mind, @Jermolene?

Reply:
Thanks @yaisog @saqimtiaz I would be inclined not to use `$tw.utils.querySelectorSafe()` for the reason given by Saq